data/SandipPalit.json

```
@@ -24,6 +24,12 @@
     }
   ],
   "links": [
+    {
+      "group": "Achievements & Recognitions",
```

[Recommendation] Since this focuses on achievements, I think it would be best to put this name in the 'Milestone' sections, and instead make this section where people can contact you.  
```suggestion
      "group": "Achievements & Recognitions",
```

---

@CBID2 this has given me an interesting feature idea, where maybe milestones could be displayed on the links section with a checkbox as it has the same fields - if you like you can start a Discussion on the repo (not issue)

---

> @CBID2 this has given me an interesting feature idea, where maybe milestones could be displayed on the links section with a checkbox as it has the same fields - if you like you can start a Discussion on the repo (not issue)

Sure no problem @eddiejaoude! :)

---

> @CBID2 this has given me an interesting feature idea, where maybe milestones could be displayed on the links section with a checkbox as it has the same fields - if you like you can start a Discussion on the repo (not issue)

Done @eddiejaoude! :) The discussion is #7692. :)